src/themes/OLH/templates/core/accounts/reset_password.html

```diff
@@ -24,10 +24,15 @@
                 <div class="callout">
                     <div class="row column log-in-form">
                         <h5>{% trans "Enter your new password twice to complete the reset process" %}:</h5>
-                        <p>{% blocktrans %}Your password should be at minimum 12 characters long. It does not need to
-                            contain specific
-                            characters but you should make it as long as possible. For more information read our
-                            <a href="#" data-open="password-modal">password guide</a>{% endblocktrans %}.</p>
+                        <p>Password Rules:</p>
```

Comment:
This text needs to be wrapped in a `{% trans %}` template tag (here and in the other themes

Reply:
This code is copied directly from the register template, you'll need to change the register template as well. Still want this change for this PR?

Reply:
Yes, that would help. With identical text the same translation will work across both pages

Reply:
OK, you'd still need to change the register template for that to work. Would you like me to add that to this PR?